keyboards/redox/rev1/rev1.h

```diff
@@ -1,10 +1,12 @@
+// Copyright 2022 xerootg
+// SPDX-License-Identifier: GPL-2.0-or-later
+
 #pragma once
 
-#include "../redox.h"
+#include "redox.h"
```

Comment:
Thats being done in redox.h, and proton_c.h is including rev1.h. If I do this, do I drop the include in proton_c.h?

Reply:
Keep both — `#pragma once` will break the apparent cycle, and this is the traditional way to ensure that everything gets included no matter which of those header files gets included first.  (Although at the current stage this is mostly future-proofing, because the main content is in `rev1.h` anyway.)